<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form</class>
 <widget class="QWidget" name="form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Power Searching</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>391</width>
     <height>111</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Google</string>
    </attribute>
    <widget class="QComboBox" name="cb_google">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="accessibleName">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>Documento</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Audio</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Video</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Digite aqui o que quer encontrar:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btGoogle">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Procurar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_entrada_google">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Google Drive</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Digite aqui o que quer encontrar:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_entrada_google_drive">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btGoogleDrive">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Procurar</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_google_drive">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Geral</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Seminarios</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Desenhos</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>jamersonwalderson@gmail.com</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_github">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../pyemail/contents/icons8-github-32.png</normaloff>../../pyemail/contents/icons8-github-32.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
